doc/classes/EditorImportPlugin.xml

```diff
@@ -1,8 +1,53 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorImportPlugin" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Used by the editor to import resources from files.
```

Comment:
The brief desc is very important. Please give an overview of the node in the brief description.
Idea (not sure exactly what the class does, just an example):
`Registers custom importers in the editor. Use the class to parse any file and store it as a new ressource type.`